utils/scripts/daos-server-user-setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -uex
```

Comment:
The above probably applies to these files as well (and the already existing utils/scripts/maven-settings.xml.in), we can then have utils/scripts being only scripts we ask installers to use and all the helper scripts for building/testing in docker will all be in utils/ci, and the helper scripts which aren't needed for docker will be in ci/

Reply:
Moved in next push.